projects/plugins/jetpack/extensions/blocks/tiled-gallery/gallery-image/edit.js

```diff
@@ -156,8 +152,8 @@ class GalleryImageEdit extends Component {
 					/>
 				</div>
 				{ /* Keep the <a> HTML structure, but ensure there is no navigation from edit */ }
-				{ href ? <a>{ img }</a> : img }
-			</figure>
+				{ href ? <a aria-hidden="true">{ img }</a> : img }
```

Comment:
Not related to your PR: I don't understand the use of the `a` tag here since the `href` value is used nowhere, nor do I understand the comment above. Do you know what would happen if we got rid of it?

Reply:
My understanding was that it relates to keeping HTML as consistent as possible with the Tiled Gallery on the front-end, which does need the a tag (being able to click on the image and open it in a new page, etc). 
Removing it doesn't create any validation errors if a Tiled Gallery is set to link to media file or attachment page from what I can see, however I also don't know if there is some other reason other than keeping the HTML parity.